.travis.yml

```diff
@@ -7,12 +7,12 @@ php:
   - hhvm
 
 env:
-  - SYMFONY_VERSION=2.6.*
+  - SYMFONY_VERSION="~2.3||~3.0"
 
 matrix:
   include:
     - php: 5.3.3
-      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest --prefer-stable"
```

Comment:
i think we had this discussion before: this will hide problems where people use an unstable old dependency. it would be better to force all dependencies that could end up with a pre-release version that does not work to require the stable version.

if there are indirect dependencies that are not listed in our composer.json but get too old, i guess we can keep this but its a risk.